### PR TITLE
zh/en: add 5.3.0 release note

### PR DESCRIPTION
**Pull request by Ehco1996**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.3 (TiDB DM 5.3 versions)
- [ ] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/releases/5.3.0.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# DM 5.3.0 Release Notes
+
+发版日期：2021 年 11 月 23 日
+
+DM 版本：5.3.0
+
+## 特别说明
+
+v2.0 到 v5.3 无兼容性变更，升级过程与正常升级无差异，仅仅是版本号上的提升。
```
Comment (Ehco1996) [inline zh/releases/5.3.0.md:13]: @Liuxiaozhen12  can you help me think of the right way to say this?

File: en/TOC.md [hunk 2]
```diff
@@ -91,6 +91,8 @@
 + [FAQ](faq.md)
 + [Glossary](glossary.md)
 + Release Notes
+  + v5.0
```
Comment (glorv) [inline en/TOC.md:94]:
```suggestion
  + v5.3
```

File: en/releases/5.3.0.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Improvements
+
+- Substantial latency reduction when Relay Log is turned on [#2225](https://github.com/pingcap/dm/pull/2225)
+- Compress/merge DML statements for incremental synchronization, significantly reducing synchronization latency [#3162](https://github.com/pingcap/ticdc/pull/3162) [#3167](https://github.com/pingcap/ticdc/pull/3167)
+- Support for DM cluster management via OpenAPI (experimental feature) [#1928](https://github.com/pingcap/dm/issues/1982)
+- Optimize the dmctl experience and add subcommands [#1746](https://github.com/pingcap/dm/pull/1746)
+- Support for stopping or pausing tasks until the transaction is finished [#1928](https://github.com/pingcap/dm/pull/1928)
+- Support for reading Relay Log files with file names greater than 999999 [#1933](https://github.com/pingcap/dm/pull/1933)
+- loader and syncer components support more monitoring metrics [#1778](https://github.com/pingcap/dm/pull/1778)
```
Comment (glorv) [inline en/releases/5.3.0.md:35]:
```suggestion
- Loader and syncer components support more monitoring metrics [#1778](https://github.com/pingcap/dm/pull/1778)
```

File: en/releases/5.3.0.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- Support for concurrent manipulation of tasks via dmctl [#1995](https://github.com/pingcap/dm/pull/1955)
+- Optimised DML concurrency for incremental synchronisation [#2043](https://github.com/pingcap/dm/pull/2043)
```
Comment (glorv) [inline en/releases/5.3.0.md:37]:
```suggestion
- Optimize DML concurrency for incremental synchronisation [#2043](https://github.com/pingcap/dm/pull/2043)
```

File: en/releases/5.3.0.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+- Prompt user when http proxy-related environment variables are detected [#1960](https://github.com/pingcap/dm/pull/1960)
+- Automatically retry when partial errors occur when reading binlog [#2014](https://github.com/pingcap/dm/pull/2014)
```
Comment (glorv) [inline en/releases/5.3.0.md:39]: remove this line, #2014 is rollbacked. 😂

Comment (Ehco1996) [reply en/releases/5.3.0.md]: removed in https://github.com/pingcap/docs-dm/pull/827/commits/1e995e646da2bd5ce374e63a6e3cbe33f112afe7

File: zh/releases/5.3.0.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- 支持通过 OpenAPI （实验特性）来运维管理 DM 集群 [#1928](https://github.com/pingcap/dm/issues/1982)
+- 优化 dmctl 的使用体验并增加一些子命令  [#1746](https://github.com/pingcap/dm/pull/1746)
+- 支持等到事务结束时才停止或暂停同步任务 [#1928](https://github.com/pingcap/dm/pull/1928)
+- 支持读取文件名大于 999999 Relay Log 文件 [#1933](https://github.com/pingcap/dm/pull/1933)
+- loader 和 syncer 组件支持更多的监控指标 [#1778](https://github.com/pingcap/dm/pull/1778)
+- 支持通过 dmctl 并发操控同步任务 [#1995](https://github.com/pingcap/dm/pull/1955)
+- 优化增量同步时 DML 并发度  [#2043](https://github.com/pingcap/dm/pull/2043)
+- 检测到 http 代理相关的环境变量时提示用户 [#1960](https://github.com/pingcap/dm/pull/1960)
+- 优化了处理 RowEvent 时的日志显示 [#2006](https://github.com/pingcap/dm/pull/2006)
+- 优化了sql 执行过慢时的日志显示 [#2024](https://github.com/pingcap/dm/pull/2024)
```
Comment (lance6716) [inline zh/releases/5.3.0.md:40]:
```suggestion
- 优化了 SQL 执行过慢时的日志显示 [#2024](https://github.com/pingcap/dm/pull/2024)
```

**Review by lance6716 — COMMENTED** [review]
lgtm for chinese part.

waiting a LGTM from doc team

File: zh/releases/5.3.0.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+## 特别说明
+
+v2.0 到 v5.3 无兼容性变更，升级过程与正常升级无差异，仅仅是版本号上的提升。
+
+## Bug 修复
+
+- 修复上下游配置 SSL 证书时高可用调度失败的问题  [#1910](https://github.com/pingcap/dm/pull/1910)
+- 修复暂停任务耗费过多时间的问题 [#1945](https://github.com/pingcap/dm/pull/1954)
+- 修复 handle-error revert 返回错误消息不明确的问题 [#1969](https://github.com/pingcap/dm/pull/1969)
+- 修复当用户使用 binlog filter 跳过某些 ddl 时同步任务失败的问题 [#1975](https://github.com/pingcap/dm/pull/1975)
```
Comment (glorv) [inline zh/releases/5.3.0.md:20]:
```suggestion
- 修复当用户使用 binlog filter 跳过某些 DDL 时同步任务失败的问题 [#1975](https://github.com/pingcap/dm/pull/1975)
```

File: zh/releases/5.3.0.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+- 开启 Relay Log 同步时大幅度降低延迟 [#2225](https://github.com/pingcap/dm/pull/2225)
+- 增量同步时压缩/合并 DML 语句，大幅度降低同步延迟 [#3162](https://github.com/pingcap/ticdc/pull/3162) [#3167](https://github.com/pingcap/ticdc/pull/3167)
+- （实验特性）支持通过 OpenAPI 来运维管理 DM 集群 [#1928](https://github.com/pingcap/dm/issues/1982)
+- 优化 dmctl 的使用体验并增加一些子命令  [#1746](https://github.com/pingcap/dm/pull/1746)
+- 支持停止或暂停同步任务时保持事务原子性 [#1928](https://github.com/pingcap/dm/pull/1928)
+- 支持读取文件名大于 999999 Relay Log 文件 [#1933](https://github.com/pingcap/dm/pull/1933)
+- Load 和 Sync 处理单元支持更多的监控指标 [#1778](https://github.com/pingcap/dm/pull/1778)
+- 支持通过 dmctl 并发操控同步任务 [#1995](https://github.com/pingcap/dm/pull/1955)
+- 优化增量同步时 DML 并发度  [#2043](https://github.com/pingcap/dm/pull/2043)
+- 检测到 http 代理相关的环境变量时提示用户 [#1960](https://github.com/pingcap/dm/pull/1960)
```
Comment (glorv) [inline zh/releases/5.3.0.md:38]:
```suggestion
- 检测到 HTTP 代理相关的环境变量时提示用户 [#1960](https://github.com/pingcap/dm/pull/1960)
```

File: zh/releases/5.3.0.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+# DM 5.3.0 Release Notes
+
+发版日期：2021 年 11 月 23 日
+
+DM 版本：5.3.0
+
+## 特别说明
+
+v2.0 到 v5.3 无兼容性变更，升级过程与正常升级无差异，仅仅是版本号上的提升。
```
Comment (Liuxiaozhen12) [inline zh/releases/5.3.0.md:13]:
```suggestion
从本次发版起，DM 改为采用 TiDB 的版本号。v2.0.x 的后一个版本号为 v5.3.0。DM v2.0.x 到 v5.3.0 无兼容性变更，升级过程与正常升级无差异，仅仅是版本号上的提升。
```

File: zh/releases/5.3.0.md (hunk 1 above)
Comment (Liuxiaozhen12) [reply zh/releases/5.3.0.md:13]: @Ehco1996 PTAL~

File: zh/releases/5.3.0.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+# DM 5.3.0 Release Notes
+
+发版日期：2021 年 11 月 23 日
+
+DM 版本：5.3.0
+
+## 特别说明
+
+从本次发版起，DM 改为采用 TiDB 的版本号。v2.0.x 的后一个版本号为 v5.3.0。DM v2.0.x 到 v5.3.0 无兼容性变更，升级过程与正常升级无差异，仅仅是版本号上的提升。
```
Comment (Liuxiaozhen12) [inline zh/releases/5.3.0.md:13]:
```suggestion
截至 DM v2.0.x，DM 采用独立于 TiDB 的版本号。DM v2.0.x 后，DM 采用与 TiDB 相同的版本号。因此，DM v2.0.x 的后一版本为 DM v5.3.0。DM v2.0.x 到 v5.3.0 无兼容性变更，升级过程与正常升级无差异，仅仅是版本号上的提升。
```

**Review by Liuxiaozhen12 — APPROVED** [review]
rest LGTM

**Pull request merged**